src/main/java/com/bmc/truesight/saas/remedy/integration/impl/GenericRemedyReader.java

```diff
@@ -160,6 +162,28 @@ public boolean login(ARServerUser arServerContext) throws RemedyLoginFailedExcep
         return payloadList;
     }
 
+    @Override
+    public boolean exceededMaxServerEntries(ARServerUser arServerContext) {
+        boolean returnVal = false;
+        List<StatusInfo> messages = arServerContext.getLastStatus();
+        if (messages != null && messages.size() > 0) {
+            List<StatusInfo> updatedMessages = new ArrayList<StatusInfo>(messages.size());
+            for (StatusInfo message : messages) {
+                if (message.getMessageNum() == ARErrors.AR_WARN_MAX_ENTRIES_SERVER) {
+                    returnVal = true;
+                    // not breaking here to handle the case of having more than one message with
+                    // AR_WARN_MAX_ENTRIES_SERVER warning. Although there is no main stream use case
+                    // but there is no much overhead of looping as well
+                } else {
+                    // Want to remove messages having warning AR_WARN_MAX_ENTRIES_SERVER from list
+                    updatedMessages.add(message);
```

Comment:
Doesn't convey what is required...

How about this;

"Adding all the warnings other than AR_WARN_MAX_ENTRIES_SERVER to the server context so that they can be ignored in the further iterations."